@remirror/react/src/components/remirror-manager.tsx

```diff
@@ -1,8 +1,13 @@
-import React, { Children, Component } from 'react';
+import React, { Children, Component, ReactNode } from 'react';
 
 import { ExtensionManager, isString, PrioritizedExtension } from '@remirror/core';
 import { baseExtensions, PlaceholderExtension } from '@remirror/core-extensions';
-import { asDefaultProps, isRemirrorExtension, RemirrorManagerProps } from '@remirror/react-utils';
+import {
+  asDefaultProps,
+  isRemirrorExtension,
+  RemirrorManagerProps,
+  isReactFragment,
```

Comment:
Linting is failing on the CI server because of this. `yarn fix` should resolve it.